freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/diagnostics/handler/DiagnosticsUpgradeTelemetryHandler.java

```diff
@@ -55,7 +56,8 @@ public void accept(Event<DiagnosticsCollectionEvent> event) {
             eventSender().sendEvent(diagnosticsCollectionEvent, event.getHeaders());
         } catch (Exception e) {
             LOGGER.debug("Diagnostics cdp-telemetry upgrade failed. resourceCrn: '{}', parameters: '{}'.", resourceCrn, parameterMap, e);
-            DiagnosticsCollectionFailureEvent failureEvent = new DiagnosticsCollectionFailureEvent(resourceId, e, resourceCrn, parameters);
+            DiagnosticsCollectionFailureEvent failureEvent = new DiagnosticsCollectionFailureEvent(resourceId, e, resourceCrn, parameters,
+                    UsageProto.CDPVMDiagnosticsFailureType.Value.UNSET.name());
```

Comment:
Is the UNSET type used here on purpose? Shouldn't it be something like "UPGRADE_TELEMETRY_FAILURE"?